Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
     capistrano-one_time_key (0.1.0)
       capistrano (~> 3.0)
     capistrano-shared_configs (0.2.2)
-    cocina-models (0.94.0)
+    cocina-models (0.94.1)
```

Comment:
I thought we would have updated this before?

Reply:
It wasn't since 0.94.1 was just released and has a fix that is needed for this code.